test-integration/client/issue-933.test.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const tap = require('tap');
+const ldapjs = require('../../lib');
+const parseDN = ldapjs.parseDN;
+
+const SCHEME = process.env.SCHEME || 'ldap';
+const HOST = process.env.HOST || '127.0.0.1';
+const PORT = process.env.PORT || 389
+const baseURL = `${SCHEME}://${HOST}:${PORT}`;
+
+const client = ldapjs.createClient({ url: baseURL });
+
+const opts = {
+   filter: '(&(objectClass=person))',
+   scope: 'sub',
+   paged: true,
+   sizeLimit: 100,
+   attributes: ['cn', 'employeeID'],
+};
+
+const baseDN = parseDN('ou=Norge Gjøvik,dc=planetexpress,dc=com');
+
+tap.test('can search OUs with Norwegian characters', (t) => {
+   client.bind(
+      'cn=admin,dc=planetexpress,dc=com',
+      'GoodNewsEveryone',
+      (err) => {
+         t.error(err, 'bind error');
+      }
+   );
+
+   client.search(baseDN.toString(), opts, (err, res) => {
```

Comment:
The test image does not contain this entry, so you're not going to get a `searchEntry` message. You'll have to add the entry before searching for it. See https://github.com/ldapjs/node-ldapjs/blob/ac588a0fadd156de27622e9dd09101e1bead5d25/test-integration/client/issue-883.test.js

Reply:
So this PR is waste? https://github.com/ldapjs/docker-test-openldap/pull/5

I just copied the same steps as you did for https://github.com/ldapjs/node-ldapjs/commit/8c9bfe81f76804e555cf2f0e2cd7b2a8524932fc

Reply:
> So this PR is waste? [ldapjs/docker-test-openldap#5](https://github.com/ldapjs/docker-test-openldap/pull/5)

I had not seen that PR yet. A note in the opening message of this PR that it requires an adjustment to the image would have been helpful.

There a couple things about this approach:

1. That PR would need to be merged and a new image built and uploaded
2. The CI configuration here would need to be updated (https://github.com/ldapjs/node-ldapjs/blob/ac588a0fadd156de27622e9dd09101e1bead5d25/.github/workflows/integration.yml#L23)

> I just copied the same steps as you did for [8c9bfe8](https://github.com/ldapjs/node-ldapjs/commit/8c9bfe81f76804e555cf2f0e2cd7b2a8524932fc)

That test follows the steps outlined above. Adding the entry to the image before writing the test here eliminates any potential issues with adding the entry via ldapjs and isolates the test to searching. It is definitely the correct approach, be we have a problem -- https://github.com/ldapjs/docker-test-openldap/issues/4.

Basically, the `master` branch in the image repo is broken because the upstream OpenLDAP implementation is broken. I have not had time to devise a solution. I will try to do so soon.

Reply:
I decided to take the easy way out and implemented a fix for the described problem in https://github.com/ldapjs/node-ldapjs/pull/935. Please address the feedback in https://github.com/ldapjs/docker-test-openldap/pull/5 and we can move this forward.